src/test/osd/osd-scrub-repair.sh

```diff
@@ -32,7 +32,18 @@ function run() {
     local dir=$1
     shift
 
-    export CEPH_MON="127.0.0.1:7107" # git grep '\<7107\>' : there must be only one
+    # Grab a random port from 7109 - 7200, this should maybe be reworked
```

Comment:
Ports 7110 and 7111 and maybe others in your 7109 - 7200 range are in use.  That's why the comment tells you to grep for any port numbers you want to use.  For example:

```
$ git grep '\<7111\>' ../src
../src/test/osd/osd-copy-from.sh:    export CEPH_MON="127.0.0.1:7111" # git grep '\<7111\>' : there must be only one
```

I'd like to see a comment line "# git grep '\<xxxx\>' : there must be only one" for each port number you are using.